### ISSUE TITLE
JPADiagram VisualizeDB Error (Netbeans 8.2)

### ISSUE THREAD

**Issue by wudongyin** [jeddict/jeddict#191]
Message : org.netbeans.db.modeler.specification.model.util.DeploymentExceptionManager.handleIntegrityException(DeploymentExceptionManager.java:223)

StackTrace : 
java.lang.NullPointerException
	at org.netbeans.db.modeler.specification.model.util.DeploymentExceptionManager.handleIntegrityException(DeploymentExceptionManager.java:223)
	at org.netbeans.db.modeler.specification.model.util.DeploymentExceptionManager.handleException(DeploymentExceptionManager.java:77)
	at org.netbeans.db.modeler.specification.model.util.DBModelerUtil.loadModelerFile(DBModelerUtil.java:176)
	at org.netbeans.modeler.core.NBModelerUtil.loadModelerFile(NBModelerUtil.java:62)
	at org.netbeans.modeler.specification.model.file.action.ModelerFileActionListener.openModelerFile(ModelerFileActionListener.java:149)
	at org.netbeans.db.modeler.specification.model.file.action.DBViewerActionListener.init(DBViewerActionListener.java:62)
	at org.netbeans.jpa.modeler.specification.model.util.DBUtil.openDBViewer(DBUtil.java:90)
	at org.netbeans.jpa.modeler.specification.model.scene.JPAModelerScene.lambda$getPopupMenuItemList$3(JPAModelerScene.java:393)
	at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
	at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2348)
	at javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
	at javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259)
	at javax.swing.AbstractButton.doClick(AbstractButton.java:376)
	at javax.swing.plaf.basic.BasicMenuItemUI.doClick(BasicMenuItemUI.java:833)
	at javax.swing.plaf.basic.BasicMenuItemUI$Handler.mouseReleased(BasicMenuItemUI.java:877)
	at java.awt.Component.processMouseEvent(Component.java:6535)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6300)
	at java.awt.Container.processEvent(Container.java:2236)
	at java.awt.Component.dispatchEventImpl(Component.java:4891)
	at java.awt.Container.dispatchEventImpl(Container.java:2294)
	at java.awt.Component.dispatchEvent(Component.java:4713)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4888)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4525)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4466)
	at java.awt.Container.dispatchEventImpl(Container.java:2280)
	at java.awt.Window.dispatchEventImpl(Window.java:2750)
	at java.awt.Component.dispatchEvent(Component.java:4713)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:159)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

ModelerFile : 
```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<jpa:entity-mappings pkg="com.mycompany.mavenproject1" v="4.1.1" rws="_14912983049482" Pws="_14912983049482" cws="_14912983049482" sm="false" xs="false" id="_14912983049481" xsi:schemaLocation="http://java.sun.com/xml/ns/persistence/orm orm_2_1.xsd" xmlns:java="http://jcp.org/en/jsr/detail?id=270" xmlns:jpa="http://java.sun.com/xml/ns/persistence/orm" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
    <jpa:entity xre="false" compositePrimaryKeyClass="UserPK" abs="false" class="User" v="true" m="false" rootElement="_14912983049481" id="_1491298312451">
        <jpa:ts e="true"/>
        <jpa:hc e="true"/>
        <jpa:eq e="true"/>
        <jpa:ath>CodeRain</jpa:ath>
        <jpa:attributes>
            <jpa:basic attribute-type="String" v="true" name="email" id="_14912983455455">
                <jpa:des>User's email</jpa:des>
                <jpa:bv>
                    <jpa:nn m="you must input an email address"/>
                </jpa:bv>
                <jpa:kbv/>
                <jpa:vbv/>
                <jpa:column name="email_address" unique="true" nullable="true" insertable="false" updatable="false" table="user" length="255"/>
            </jpa:basic>
            <jpa:one-to-many orp="false" collection-type="java.util.List" connected-entity-id="_1491298670582" connected-attribute-id="_149129992894824" v="true" name="posts" id="_149129992894925">
                <jpa:bv/>
                <jpa:kbv/>
                <jpa:vbv/>
            </jpa:one-to-many>
            <jpa:transient attribute-type="String" v="true" name="password" id="_149129889344514">
                <jpa:bv/>
                <jpa:kbv/>
                <jpa:vbv/>
            </jpa:transient>
            <jpa:id attribute-type="Long" v="true" name="id" id="_14912983124983">
                <jpa:bv/>
                <jpa:kbv/>
                <jpa:vbv/>
                <jpa:column name="user_id" unique="false" nullable="true" insertable="true" updatable="true"/>
                <jpa:generated-value strategy="AUTO"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:nq e="false" name="User.findAll">
            <jpa:query>Select e from User e</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="User.findByEmail">
            <jpa:query>Select u from User u where u.email=:email</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="User.findByPassword">
            <jpa:query>Select u from User u where u.password=:password</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="User.findByPosts">
            <jpa:query>Select u from User u where u.posts=:posts</jpa:query>
        </jpa:nq>
        <jpa:el/>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="PostPK" abs="false" class="Post" v="true" m="false" rootElement="_14912983049481" id="_1491298670582">
        <jpa:ts e="true"/>
        <jpa:hc e="true"/>
        <jpa:eq e="true"/>
        <jpa:ath>CodeRain</jpa:ath>
        <jpa:attributes>
            <jpa:basic attribute-type="String" v="true" name="content" id="_149129870937012">
                <jpa:bv/>
                <jpa:kbv/>
                <jpa:vbv/>
            </jpa:basic>
            <jpa:many-to-one optional="true" primaryKey="false" connected-entity-id="_1491298312451" connected-attribute-id="_149129992894925" v="true" name="user" id="_149129992894824">
                <jpa:bv/>
                <jpa:kbv/>
                <jpa:vbv/>
            </jpa:many-to-one>
            <jpa:id attribute-type="Long" v="true" name="id" id="_149129867058210">
                <jpa:bv/>
                <jpa:kbv/>
                <jpa:vbv/>
                <jpa:column name="post_id" unique="false" nullable="true" insertable="true" updatable="true"/>
                <jpa:generated-value strategy="AUTO"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:nq e="false" name="Post.findAll">
            <jpa:query>Select e from Post e</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Post.findByContent">
            <jpa:query>Select p from Post p where p.content=:content</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Post.findByAuthor">
            <jpa:query>Select p from Post p where p.author=:author</jpa:query>
        </jpa:nq>
        <jpa:el/>
    </jpa:entity>
    <jpa:snp loc="BEFORE_PACKAGE" e="true">/**
 * This file was generated by the Jeddict
 */</jpa:snp>
    <jpa:c>
        <ct>java.util.List</ct>
        <ct>java.util.Set</ct>
        <ct>java.util.Map</ct>
        <ct>java.util.Collection</ct>
    </jpa:c>
    <jpa:wsl>
        <jpa:ws n="Main" id="_14912983049482">
            <i ref="_1491298312451" x="72" y="32">
                <el/>
            </i>
            <i ref="_1491298670582" x="368" y="32">
                <el/>
            </i>
        </jpa:ws>
    </jpa:wsl>
</jpa:entity-mappings>
```

**Comment by jGauravGupta** [comment]
Hi @wudongyin ,
Issue will be fixed.
ATM just remove the table name "user" from  User entity > email attribute > property panle > column properties > Table name

**Comment by jGauravGupta** [comment]
Seems to be fixed with current version 5.1 by showing a error message with details.

![image](https://user-images.githubusercontent.com/15934072/45584017-ee515b00-b8e9-11e8-8900-b74540f403be.png)
